samza-core/src/main/java/org/apache/samza/task/EpochTimeScheduler.java

```diff
@@ -81,7 +81,7 @@ private EpochTimeScheduler(ScheduledExecutorService executor) {
     }
   }
 
-  void registerListener(TimerListener listener) {
+  public void registerListener(TimerListener listener) {
```

Comment:
Should this class be in container instead of task as well?

Reply:
I noticed we have a scheduler package and CallbackSchedulerImpl is the main consumer of this class. 
Ended up moving it there. Let me know if you think we consolidate scheduler and container package.